lib/embed/embed.dart

```diff
@@ -96,6 +96,25 @@ class PlaygroundMobile {
     });
   }
 
+  /**
+   * Return if strong mode should be enabled
+   */
+  bool _isStrongMode(String strong) {
```

Comment:
This is populating the UI from the query param, but I don't see where we're pulling that out and passing that to analysis?